Makefile

```diff
@@ -158,4 +160,13 @@ nightlybuild:
 	$(ECHO_V)rm -rf *trie*.db
 	$(GOBUILD) -o ./bin/$(BUILD_TARGET_MINICLUSTER) -v ./tools/minicluster
 	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib
-	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
\ No newline at end of file
+	./bin/$(BUILD_TARGET_MINICLUSTER) -timeout=14400
+
+.PHONY: recover
+recover:
+	$(ECHO_V)rm -rf ./e2etest/*chain*.db
+	$(GOBUILD) -o ./bin/$(BUILD_TARGET_RECOVER) -v ./tools/staterecoverer
+	export LD_LIBRARY_PATH=$(LD_LIBRARY_PATH):$(PWD)/crypto/lib
+	./bin/$(BUILD_TARGET_RECOVER) -config-path=e2etest/config_local_delegate.yaml
```

Comment:
why we use the config in e2etest?

Reply:
config_local_delegate.yaml should be moved out from e2etest since it is used for many purposes. Can have a sampleconfigs package to hold different configs. Could defer to another PR.

Reply:
This config corresponds to the one in "make reboot" and "make run".